portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,17 +26,28 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-    
+static ArrayList<String> myarraylist = new ArrayList<String>(); 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    ArrayList<String> myarraylist = new ArrayList<String>();
-    myarraylist.add("One");
-    myarraylist.add("Two");
-    myarraylist.add("Three");
-   
+
     Gson gson = new Gson();
     String json = gson.toJson(myarraylist);
-    response.setContentType("application/json;");
+    response.setContentType("application/json");
     response.getWriter().println(json);
+ }
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
nit: Please fix the indentation of this line. It should start from the first column.